### PR TITLE
Blog Page

### PR DESCRIPTION
**Pull request by alexgirard**
Blog Page:
- added `testPosts` for now, not sure the starting content we want
- also took creative liberty with the links and hover states, feel free to change anything if you have other ideas

**Desktop**

Screen Shot 2020-08-02 at 4 43 19 PM

**Mobile**
Screen Shot 2020-08-02 at 4 44 49 PM

**Hover States**
| Screen Shot 2020-08-02 at 4 43 35 PM | Screen Shot 2020-08-02 at 4 43 43 PM |

### CODE REVIEW

**Comment by RobbieZhuang** [comment]
AWW I love it!! The hover states look so good 😍

Also, to get rid of lint errors, u can run ‘npm run format’, thanks!

I’ll make a review later today :)

**Comment by RobbieZhuang** [comment]
Also gonna loop in @yayimahuman to do an optional review if he has the time :)

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "dependencies": {
         "ajv": "^6.10.0",
         "bootstrap": "^4.5.0",
+        "lodash": "^4.17.19",
```
[the remark this answers is not in the document]
Comment (alexgirard) [reply package.json:18]: ahaha I was using yarn 🙃

**Comment by RobbieZhuang** [comment]
Feel free to fix conflicts & merge whenever 🎉

**Comment by RobbieZhuang** [comment]
LGTM, Thank you!! 🎉🎉

**Pull request merged**